tests/components/template/test_lock.py

```diff
@@ -1,280 +1,163 @@
 """The tests for the Template lock platform."""
-import logging
+import pytest
 
 from homeassistant import setup
 from homeassistant.components import lock
 from homeassistant.const import ATTR_ENTITY_ID, STATE_OFF, STATE_ON, STATE_UNAVAILABLE
-from homeassistant.core import callback
 
-from tests.common import assert_setup_component, get_test_home_assistant
+from tests.common import assert_setup_component, async_mock_service
 
-_LOGGER = logging.getLogger(__name__)
 
+@pytest.fixture
+def calls(hass):
+    """Track calls to a mock service."""
+    return async_mock_service(hass, "test", "automation")
 
-class TestTemplateLock:
-    """Test the Template lock."""
 
-    hass = None
-    calls = None
-    # pylint: disable=invalid-name
+async def test_template_state(hass):
+    """Test template."""
+    with assert_setup_component(1, lock.DOMAIN):
+        assert await setup.async_setup_component(
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6333224)